client/components/Dashboard/RecentScenarios.jsx

```diff
@@ -0,0 +1,69 @@
+import './Dashboard.css';
+
+import { Button, Icon } from '@components/UI';
+import { Container, Divider, Header, List, Segment } from '@components/UI';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import ScenarioCard from '@components/ScenariosList/ScenarioCard';
+import { getRecentScenarios } from '@actions/scenario';
+
+const ScenarioContent = ({ scenarios = [] }) => {
+  if (scenarios.length) {
+    return (
+      <List className="dashboard-grid">
+        {scenarios.map((scenario, index) => {
+          return (
+            <List.Item key={`scenario-card-${scenario.id}-${index}`}>
+              <ScenarioCard scenario={scenario} />
+            </List.Item>
+          );
+        })}
+      </List>
+    );
+  } else {
+    return (
+      <Segment secondary padded className="dashboard-empty">
+        <p>
+          No scenarios created. <a href="#">Create a new scenario.</a>
+        </p>
+      </Segment>
+    );
+  }
+};
+
+const RecentScenarios = () => {
+  const dispatch = useDispatch();
+  const scenarios = useSelector(state => state.recentScenarios);
+
+  useEffect(() => {
+    dispatch(getRecentScenarios());
+  }, [dispatch]);
+
+  return (
+    <Container fluid id="recent-scenarios">
+      <Header className="dashboard-subheader">
+        <Header as="h2">Your most recent scenarios</Header>
+        <Button
+          icon
+          primary
+          labelPosition="left"
+          name="Create a new cohort"
+          size="small"
+          href="/"
+          as="a"
+        >
+          <Icon name="add" />
+          Create a new scenario
+        </Button>
+        <Button size="small" href="/scenarios">
+          View all scenarios
+        </Button>
+      </Header>
+      <Divider />
+      <ScenarioContent scenarios={scenarios} />
```

Comment:
This is probably where `<ScenarioDetailModal />` should go, and the state for `open` and `selected` scenario will be managed inside `RecentScenarios` by sending some handler, ie. 

```js
<ScenarioContent 
  onScenarioSelectionClick={some state updating handler} 
  scenarios={scenarios} 
/>
```